components/MemberList/index.js

```diff
@@ -41,6 +46,12 @@ const Table = ({ members }) => {
           >
             Name
           </th>
+          <th
```

Comment:
we can make a TH component in this file and use it as `<TH></TH>` so we do not have to repeat the styling!